storage/backend/boltoption_unix.go

```diff
@@ -30,5 +30,5 @@ import (
 // silently ignore this flag. Please update your kernel to prevent this.
 var boltOpenOptions = &bolt.Options{
 	MmapFlags:       syscall.MAP_POPULATE,
-	InitialMmapSize: InitialMmapSize,
+	InitialMmapSize: int(InitialMmapSize),
```

Comment:
Yes, probably. See here: https://github.com/boltdb/bolt/blob/master/bolt_arm.go#L4. 
I would prefer moving this to a specific ARM file.